### DISCUSSION TITLE
Adjusting scoreDisplayMode of audit result using config variable

### DISCUSSION THREAD

**Discussion by GotGud2** [GoogleChrome/lighthouse#15673] [Q&A]
The change from version 11.2.0 to version 11.3.0 changed f.e. the scoreDisplayMode of the server-response-time.js audit result from METRIC_SAVINGS to INFORMATIVE

See changes made in #15447 to server-response-time.js

This change nullifies the audits score.
audit.js line 324 - 328
```javascript

  static _normalizeAuditScore(score, scoreDisplayMode, auditId) {
    if (scoreDisplayMode !== Audit.SCORING_MODES.BINARY &&
        scoreDisplayMode !== Audit.SCORING_MODES.NUMERIC &&
        scoreDisplayMode !== Audit.SCORING_MODES.METRIC_SAVINGS) {
      return null;
    }
    /* ... */
``` 

Currently, there seems to be no way to set the scoreDisplayMode back to METRIC_SAVINGS using the default-config.js because the code doesn't use a configurable variable but instead uses constants hardcoded into the audits return.

server-response-time.js line 95
```javascript

    return {
     /* ... */
      scoreDisplayMode: passed ? Audit.SCORING_MODES.INFORMATIVE : undefined,
     /* ... */
    };
``` 

Am I missing something or is this functionality currently missing?

**Comment by adamraine** [comment]
I don't think it's possible to do what you want with the config. Can you describe the use case a little more?

**Reply by GotGud2** [reply]
We need the server-response-time audit with its score for our internal QM. How can we achieve this without cloning/extending the audit and changing the clones scoreDisplayMode back to METRIC SAVINGS?

**Reply by adamraine** [reply]
If I am understanding your use case correctly, the score is `null` when the audit becomes informative and this is difficult to interpret from a QM perspective.

In this case, the audit is informative because the server response time is good but we still want to surface the it in the report. From a QM perspective, informative audits can be interpreted the same way as passing audits in that the problem (server response time in this case) is not bad enough to to be flagged. To put it another way, if the audit is informative it would have had a score of 1 under the old system (prior to the changes in 11.3). Again there isn't a way to change the scoring methodology right now but that's how you can interpret the score.

This post does highlight how the new scoring system can be confusing though for audits that change their score display mode so we can discuss ways of improving.

**Reply by adamraine** [reply]
https://github.com/GoogleChrome/lighthouse/issues/15686
